SPEC.md

```diff
@@ -111,11 +134,13 @@ The body may contain arbitrary data. Headers should specify content type and enc
 
   **Body**: Arbitrary data conveying the operation's result. Headers should specify content type and encoding.
 
-- `201 Created`: Operation was started and will complete asynchronously.
+- `201 Created`: Operation was started and will complete asynchronously. It may return `Nexus-Link` headers that
+  contain information about the operations done by the handler.
```

Comment:
```suggestion
- `201 Created`: Operation was started and will complete asynchronously. It may return `Nexus-Link` headers to
  associate resources to this operation.
```